### ISSUE TITLE
Install weechat-matrix in virtualenv

### ISSUE THREAD

**Issue by non-Jedi** [poljar/weechat-matrix#105]
Hi poljar. I'd really like to not pollute my globally installed packages with the weechat-matrix dependencies. Is there any way to install them in a virtualenv instead? I'm thinking not since I guess weechat just directly executes the main.py file, but maybe you know a way? If so, this issue is a request to document such in the README.

**Comment by trygveaa** [comment]
Yeah, it's possible. Either activate the virtualenv before starting weechat. Or, if weechat is already running, you can add this after the `__future__` import in `main.py`:

```
import sys
sys.path.insert(0, '/home/trygve/dev/weechat-venv/lib/python2.7/site-packages')
```

Replace that path with the one to your virtualenv.

**Comment by matthijskooijman** [comment]
I created a PR to do this automatically at #117. Note that I took a slightly different route than modifying `sys.path` directly. I used the `activate_this.py` script, which is the [official way]( https://virtualenv.pypa.io/en/latest/userguide/#using-virtualenv-without-bin-python) to activate a virtualenv from python (I noticed this existed when I saw [weeplug](https://github.com/pyroscope/weeplug/blob/82663c1c6b96d47e664af25e7ac6dfebb6525293/src/weeplug/data/shim.py#L15-L16) did it).

This has the advantage that is sets up the environment completely (including interpreting .pth files using the `site` python package) and does not need hardcoding the path to `site-packages` (which is apparently different between different Python runtimes and operating systems). The downside is that pyvenv and `python3 -m venv` are note supported this way, but they do not seem to have any documented way to be activated from within Python.

We could support python3 venv and pyvenv by manually activating the environment (or maybe just modifying `sys.path`), for example by simply copying the contents of `activate_this.py`, but that seemed a bit too complicated (and I had no way of testing this right now): https://github.com/pypa/virtualenv/blob/master/virtualenv_embedded/activate_this.py
